### DISCUSSION TITLE
Práctica CSS

### DISCUSSION THREAD

**Discussion by GerardoSerrato** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#412] [Show and tell]
Me había atrasado bastante pero aquí está mi página terminada. 
No es responsive, será el siguiente paso en mi aprendizaje.
Les agradecería cualquier comentario para mejorar.

[Página clon](https://gerardoserrato.github.io/index.html)

**Comment by Bestoff-77** [comment]
Very nice !

**Comment by MarcosSilvaG** [comment]
excelente :D. 
Un consejo (no lo pediste pero puede funcionar) checa la pseudo-clase [hover](https://developer.mozilla.org/es/docs/Web/CSS/:hover), es "fácil" de implementar y le da más movimiento a la página. Saludos 🚀🚀

**Reply by GerardoSerrato** [reply]
Muchas gracias por el consejo!!
